hw/ip/kmac/data/kmac.hjson

```diff
@@ -98,6 +98,18 @@
       desc:    "Number of words for Encoded NsPrefix."
       local:   "true"
     }
+    { name:    "NumEntriesMsgFifo"
+      type:    "int"
+      default: "10"
+      desc:    "Number of entries in the message FIFO."
```

Comment:
I would add a second sentence here to clarify that this needs to be equal to `kmac_pkg::MsgFifoDepth`.